src/Language/PureScript/Docs/RenderedCode/Render.hs

```diff
@@ -0,0 +1,172 @@
+
+-- | Functions for producing RenderedCode values from PureScript Type values.
+
+module Language.PureScript.Docs.RenderedCode.Render (
```

Comment:
Just a word of warning - `pattern-arrows` will spin if a case isn't handled, effectively adding parens forever, so you'll want to make sure your base cases are exhaustive :)